package.json

```diff
@@ -1,16 +1,24 @@
 {
   "name": "swl-builder",
+  "homepage": "https://lukezors.github.io/swl-builder",
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "enzyme": "^3.10.0",
+    "enzyme-adapter-react-16": "^1.15.1",
+    "gh-pages": "^2.1.1",
+    "prop-types": "^15.7.2",
     "react": "^16.11.0",
     "react-dom": "^16.11.0",
-    "react-scripts": "3.2.0"
+    "react-scripts": "3.2.0",
+    "styled-components": "^4.4.1"
```

Comment:
Might be a good time to throw a linter up in here! Here's a link to my .eslintrc if you want it - airbnb's lint is a great place to start, and I've excluded a few rules I find useless: https://github.com/jksaunders/bonfire/blob/master/.eslintrc

Using airbnb's lint is pretty educational, 10/10 recommend 👌

Reply:
Looks like create app has eslint built in s I added a script, not sure if you can override the rc or not